Content.Client/Clothing/ClientClothingSystem.cs

```diff
@@ -331,7 +331,7 @@ private void RenderEquipment(EntityUid equipee, EntityUid equipment, string slot
             if (displacementData is not null)
             {
                 //Checking that the state is not tied to the current race. In this case we don't need to use the displacement maps.
-                if (layerData.State is not null && inventory.SpeciesId is not null && layerData.State.EndsWith(inventory.SpeciesId))
+                if (inventory.DisplaceSpeciesAppropriateClothing == false && layerData.State is not null && inventory.SpeciesId is not null && layerData.State.EndsWith(inventory.SpeciesId))
```

Comment:
```suggestion
                if (!inventory.DisplaceSpeciesAppropriateClothing && layerData.State is not null && inventory.SpeciesId is not null && layerData.State.EndsWith(inventory.SpeciesId))
```
It's a bool, you can just invert it.

Reply:
[Done](https://github.com/space-wizards/space-station-14/pull/39361/commits/4d1a8e4e04a45cf97e37a55d108a7b6bc546387a)